boards/nordic/nrf54h20dk/nrf54h20dk_nrf54h20-memory_map.dtsi

```diff
@@ -187,8 +187,8 @@
 		#address-cells = <1>;
 		#size-cells = <1>;
 
-		storage_partition: partition@136000 {
-			reg = <0x136000 DT_SIZE_K(24)>;
+		storage_partition: partition@1fa000 {
```

Comment:
This partition must end before 0x0e1e9000, because Secure Domain's storage starts there

Reply:
I would say in a WiFi shield overlay you should apply the minimal set of changes to make WiFi running in most simple WiFi samples. I.e. shrink the DFU partition, move the PPR partition (not remove), expand the cpuapp partition.

If complex samples or applications like Matter require even more memory, the sample overlay should do even more modifications (e.g. remove ppr and dfu partitions and move them to external memory).

@ArekBalysNordic, is this consistent with your needs?

Reply:
Perhaps placing PPR code in front of the application code or in front of application NVM data would help to achieve this

Reply:
Actually, in Matter samples, we need to modify partitions to our purposes because we need as much free memory for customers as possible. So I guess, no matter which will be set in the global memory map we need to modify both RAM and MRAM in all samples. But I agree that removing PPR, or removing whatever should be done within the samples DTS file instead of shield one. It looks like the WiFi shield overlay allows us to run Matter over WiFi thanks to extending RAM and MRAM, but we need to modify partitions anyway because otherwise, it will be incompatible with DFU.

Reply:
Thanks, I will then proceed with moving PPR code to the front.

Reply:
@hubertmis : Correct. The memory range `0xe1e9000 - 0xe1f0000` is reserved for SDFW.